lib/livebook/live_markdown/export.ex

```diff
@@ -52,6 +56,26 @@ defmodule Livebook.LiveMarkdown.Export do
     data
   end
 
+  # Handle old flow for backward compatibility with Kino <= 0.10.0
```

Comment:
Do we want to add a TODO to eventually remove it?

Reply:
I wasn't sure, but it's not critical, worst case we just don't export the output when someone uses an old kino. I added a TODO to remove on livebook v0.13, which should be enough time :)